module/Photo/view/partial/tags-new.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+$this->scriptUrl()->requireUrl('photo/photo/tag/remove', ['photo_id', 'lidnr'])
+                  ->requireUrl('member/view', ['lidnr']);
+?>
+<?php if($this->acl('photo_acl')->isAllowed('tag', 'view')): ?>
+    <strong><?= $this->photo->getAlbum()->getName() ?></strong>
+    <?php $tagCount = $this->photo->getTags()->count(); $i = 1; ?>
+    <?= $tagCount ? $this->translate(', in this photo:') : '' ?>
+    <div class="tagList tagList-<?= $this->photo->getId() ?>">
```

Comment:
A nice-to-have is if the tag list could be hidden. On smaller screens it's a little bit obtrusive.

[Screenshot](https://user-images.githubusercontent.com/706385/65905776-7bfecc00-e3c1-11e9-895c-3bf8a165ca8d.jpeg)

Reply:
You can hide them by tapping on the center of your screen. It follows the default behaviour of mobile photo viewers.

Reply:
Hmm. On my device tapping on the center of the screen only zooms-in the picture.

Reply:
I experience the same issue as Armand.